EmojiArt/Document/EmojiArtViewModel.swift

```diff
@@ -68,18 +66,26 @@ class EmojiArtViewModel: ObservableObject {
     }
     
     func moveEmoji(_ emoji: EmojiArtModel.Emoji, by offset: CGSize) {
-        if let index = emojiArtModel.emojis.index(matching: emoji) {
-            emojiArtModel.emojis[index].x += Int(offset.width)
-            emojiArtModel.emojis[index].y += Int(offset.height)
+        guard let index = emojiArtModel.emojis.index(matching: emoji) else {
+            return
         }
+        
+        debugPrint("EMOJI IN MOOVE: ", emoji.x, emoji.y, offset)
```

Comment:
I recommend to remove debugPrints when debug is over and it's not necessary anymore